edl/sgx/sgx_attestation.edl

```diff
@@ -4,7 +4,7 @@
 /*
 **==============================================================================
 **
-** sgx/attestation.edl:
+** sgx/sgx_attestation.edl:
```

Comment:
Does EDL actually care about this? we don't qualify other files in this folder the same way, and I suspect that the edl/attestation.edl file will be the one going away in the long run.

Reply:
It depends on how you set your edl include path in oeedger8r. If you pass both `edl` and `edl/sgx` as include paths, then there will be a conflict (attestation.edl exists twice) and I believe oeedger8r will use the second one. I figured it was easier to just rename it to avoid conflict. Especially since as you say this edl file is likely to go away in the future.

Reply:
Actually this EDL file (edl/sgx/attestation.edl) is likely to hang around because of `oe_get_quote_verification_collateral_ocall()`. My comment is that edl/attestation.edl is likely to go away because it services the host-side call-in to `oe_get_report()`, which should ultimately go away in favor of the pluggable attestation APIs.